### PR TITLE
Documentation for creating events

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: docs/book/basics/creating_events.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{% endpanel %}
+
+{% method %}
+
+Events are published from a controller using an event recorder, which is automatically configured
+by `kubebuilder create resource`.  The event recorded is intended to be used from the `Reconcile`
+function.
+
+```go
+Event(object runtime.Object, eventtype, reason, message string)
```
Comment (Liujingfang1) [inline docs/book/basics/creating_events.md]: Can you replace `eventtype` by `type` to be consistent with subsequent lines?

**Comment by droot** [comment]
Will it make sense to include link to the recorder interface --> https://github.com/kubernetes/client-go/blob/master/tools/record/event.go#L56 so that user can discover all the three methods ?

File: docs/book/basics/creating_events.md (hunk 1 above)
Comment (pwittrock) [reply docs/book/basics/creating_events.md]: Done

**Comment by pwittrock** [comment]
@droot Added

File: docs/book/basics/creating_events.md (hunk 1 above)
Comment (pwittrock) [reply docs/book/basics/creating_events.md]: type is a reserved word, so changing the doc to eventtype

**Pull request merged**